pkg/server/spec/deploy.go

```diff
@@ -112,6 +122,16 @@ func (b *DeployBuilder) Build() *Deploy {
 			PreStop: &PreStop{DrainTimeoutSeconds: defaultDrainTimeoutSeconds},
 		}
 	}
+
+	if b.d.DNSConfig == nil {
+		b.d.DNSConfig = &DNSConfig{
+			Options: append([]DNSOptions{}, DNSOptions{
+				Name:  "ndots",
+				Value: "2",
```

Comment:
what about creating a config (with that values as default) to hold this configuration?

Reply:
@drgarcia1986 that was my first idea, then I kept hard coded 'cause ndots 2 is the new default value on CoreDNS. I will try to create this config.

Reply:
You can follow your heart